### DISCUSSION TITLE
Evaluate similarity of two images for deduping

### DISCUSSION THREAD

**Discussion by chukinas** [ImageMagick/ImageMagick#5712] [Help]
I want to compare two images and determine if they look the same. By same I mean they have the same aspect ration and look the same to the human eye. But maybe one is a resized or compressed version of the other. 

How would I go about this?

**Comment by fmw42** [comment]
See compare -metric phash

http://www.fmwconcepts.com/misc_tests/perceptual_hash_test_results_510/index.html
